shotover-proxy/tests/test-topologies/tee/subchain_with_mismatch.yaml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Why is the mismatch case the one tested against a real database instance?
Shouldnt we be testing the match case against a real database to make sure it matches unless something bad happens?
And then for the mismatch case we want to simulate something bad happening so thats where use the returner.

Reply:
Not sure how else to test that the subchain runs other than put a redis cluster at the end and check a value was sent to it. 

We can test the match case with a real database too though.

Reply:
Well at the very least we could use a single redis instance rather than a cluster which will be faster to startup.

Possible improvements could be inspecting the log and using the log transform
Or we could create a transform that writes the message to the file system. Then we could be bank scale :laughing: 

But lets just use a single redis instance for now.
Maybe leave a comment explaining why we need a redis instance here.